test/cli_save_and_load_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+
+	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+// PouchSaveLoadSuite is the test suite for save and load CLI.
+type PouchSaveLoadSuite struct{}
+
+func init() {
+	check.Suite(&PouchSaveLoadSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchSaveLoadSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	environment.PruneAllContainers(apiClient)
+}
+
+// TestSaveLoadWorks tests "pouch save" and "pouch load" work.
+func (suite *PouchSaveLoadSuite) TestSaveLoadWorks(c *check.C) {
+	res := command.PouchRun("pull", busyboxImage)
+	res.Assert(c, icmd.Success)
+
+	res = command.PouchRun("image", "inspect", busyboxImage)
+	res.Assert(c, icmd.Success)
+
+	before := []types.ImageInfo{}
+	if err := json.Unmarshal([]byte(res.Stdout()), &before); err != nil {
+		c.Errorf("failed to decode inspect output: %v", err)
```

Comment:
I think here we should use `c.Fatalf` rather than `c.Errorf`? WDYT? @xiechengsheng